app/static/css/_about.scss

```diff
@@ -79,11 +79,47 @@
     }
 }
 
+.dataone_repos {
+    text-align: center;
+    margin: 0.25rem;
+    padding: 0.25rem;
+    display: inline-block;
+
+    &:hover {
+        -moz-box-shadow: 0 0 2px 2px constants.$light-blue;
+        -webkit-box-shadow: 0 0 2px 2px constants.$light-blue;
+    }
+
+    @include constants.breakpoint(medium-and-up) {
+        margin: 1rem;
+        padding: 1rem;
+    }
+}
+
+.bas {
+
+    background-color: black;
```

Comment:
I know this is really persnickety, but please use constants.$black if you're going to use black.